### PR TITLE
Add missing functionalities like disableImageLoading, choose requests…

### PR DESCRIPTION
**Pull request by fguery**
… to stop, image capture, step timeout

### CODE REVIEW

File: src/Driver.php [hunk 1]
```diff
@@ -194,6 +194,97 @@ public function setHeaders(array $headers)
...
+     */
+    public function setRequestsToSkip($urlJsRegexes)
+    {
+        $this->script .= '
+casper.options.onResourceRequested = function(casper, requestData, request) {';
+
+        foreach ($urlJsRegexes as $regex) {
+            $this->script .= '
+    if ((' . $regex . ').test(requestData.url)) {
+        console.log("SKIPPING" + requestData.url);
```
Comment (Putzy) [inline src/Driver.php]: Most minor thing ever - missing a space here before the URL.

**Comment by Putzy** [comment]
Apart from that, 👍 Interested to see what else you're planning.......

File: src/Driver.php (hunk 1 above)
Comment (fguery) [inline src/Driver.php]: http://www.nooooooooooooooo.com

**Comment by Putzy** [comment]
👍

**Pull request merged**